### ISSUE TITLE
Scrollbar flashes on Firefox

### ISSUE THREAD

**Issue by pdugan20** [jossmac/react-toast-notifications#114]
For single line toasts on Firefox, the scrollbar briefly appears when entering, and exiting.
![1](https://user-images.githubusercontent.com/1837100/83973236-d7ab1e80-a899-11ea-8b64-a66b41664fee.gif)

I can fix this by overriding the class manually with:
```
.react-toast-notifications__container {
    overflow-y: hidden !important;
}
```

It looks like there have been some issues filed here in the past? Looks like it behaves as expected in Safari and Chrome.

**Comment by minimaluminium** [comment]
Seems like the issue was fixed by #79, but an older version was installed with npm. Maybe the npm version is outdated?

**Comment by pdugan20** [comment]
Seeing this on 2.4.0. Running Firefox on Mac.

**Comment by sebastianserrano** [comment]
Same issue here on package version 2.4.0 and Firefox 77.0.1 (64-bit) on Mac

**Comment by s-h-a-d-o-w** [comment]
It also happens on https://jossmac.github.io/react-toast-notifications/ (Chrome 84.0.4147.105)

![Peek 2020-08-11 21-24](https://user-images.githubusercontent.com/16936908/89940013-2a49e380-dc19-11ea-96f7-fd9abaff2c30.gif)

**Comment by ngc2392** [comment]
Anyone having this issue on Chrome? I'm seeing the scrollbars when I position the toasts 'bottom-center'. I'm using Version 84.0.4147.125 (64-bit) on Mac.

**Comment by DaleyKD** [comment]
I see this issue on https://jossmac.github.io/react-toast-notifications/ with Chrome 87.  Turn off auto-dismiss, add quite a few, then delete the last one.

I THINK? I can work around the issue for my needs by setting the CSS of the `css-3y3n1b` (I know it's random) to include `bottom: 0`

**Comment by jossmac** [comment]
@DaleyKD avoid targeting generated class names, they will change.

**Comment by DaleyKD** [comment]
Yeah, I know. Was just using it as an example. I was just messing around on
the example site without getting into the code yet.

Thanks for the good looking out, though!

**Comment by jossmac** [comment]
Resolved in `2.4.1`
